src/lib/cinchy.service.ts

```diff
@@ -241,171 +269,171 @@ export class CinchyService {
             });
         }
         let apiUrl = this.cinchyRootUrl + '/API/ExecuteCQL';
-        let errorMsg = 'Failed to execute query ' + query;
+        let errorMsg = '[EDIT] Failed to execute query ' + query;
```

Comment:
@DAquilinaCinchy What's "[EDIT]"?

Reply:
It's a mistake that has now been corrected